api/src/pcapi/core/mails/transactional/bookings/booking_cancellation.py

```diff
@@ -21,6 +21,7 @@
 def send_booking_cancellation_emails_to_user_and_offerer(
     booking: Booking,
     reason: BookingCancellationReasons | None,
+    rejected: bool | None = False,
```

Comment:
Je ne vois aucun cas dans lequel on a `None`
```suggestion
    rejected: bool = False,
```

Reply:
booking_cancellation ligne 24
`    rejected: bool = False,`
FAIT